gammapy/cube/tests/test_models.py

```diff
@@ -82,21 +82,29 @@ def diffuse_evaluator(diffuse_model, exposure, background, psf, edisp):
     return MapEvaluator(diffuse_model, exposure, background, psf=psf, edisp=edisp)
 
 
+def test_BackgroundModel(background):
+    bkg1 = BackgroundModel(background, norm=2.0).evaluate()
+    assert_allclose(bkg1[0][0][0], background.data[0][0][0] * 2.0, rtol=1e-3)
+    assert_allclose(bkg1.sum(), background.data.sum()*2.0, rtol=1e-3)
+
+    bkg2 = BackgroundModel(background, norm=2.0, tilt=0.2).evaluate()
+    assert_allclose(bkg2[0][0][0], 2.254e-07, rtol=1e-3)
+    assert_allclose(bkg2.sum(), 7.352e-06, rtol=1e-3)
+
+
 class TestSkyModels:
-    def setup(self):
-        self.sky_models = SkyModels([sky_model(), sky_model()])
 
-    def test_to_compound_model(self):
-        sky_models = self.sky_models
+    def test_to_compound_model(self, sky_model):
+        sky_models = SkyModels([sky_model, sky_model])
```

Comment:
I solved the issues with the latest pytest version in #1976. Would you mind reverting you changes here and just keep the `test_background_model` function? Once #1976 is merged the tests will pass again...

Reply:
Thanks! I wasted a lot of time on this without seeing your PR

Reply:
I'm sorry that this blocked your work. Of course you can contact me any time on Slack if you  encounter a problem of this kind, just to prevent that you waist your time on it again.